packages/dx-react-chart/src/plugins/pane-layout.tsx

```diff
@@ -36,6 +39,8 @@ export class PaneLayout extends React.PureComponent {
             <TemplateConnector>
               {({ layouts }, { changeBBox }) => {
                 const { width, height } = layouts.pane;
+                const clipPathId = `clip_path_${numDefs}`;
```

Comment:
Unique clip path is bound to a component instance not to a particular template render occurrence.

Reply:
I meant - *should be* bound.